chain-signatures/node/src/protocol/presignature.rs

```diff
@@ -23,12 +27,94 @@ use near_account_id::AccountId;
 pub type PresignatureId = u64;
 
 /// A completed presignature.
+#[derive(Clone)]
 pub struct Presignature {
     pub id: PresignatureId,
     pub output: PresignOutput<Secp256k1>,
     pub participants: Vec<Participant>,
 }
 
+impl Default for Presignature {
+    fn default() -> Self {
+        Self {
+            id: 1,
+            output: PresignOutput {
+                big_r: <Secp256k1 as CurveArithmetic>::AffinePoint::default(),
+                k: <Secp256k1 as CurveArithmetic>::Scalar::ZERO,
+                sigma: <Secp256k1 as CurveArithmetic>::Scalar::ONE,
+            },
+            participants: vec![Participant::from(1), Participant::from(2)],
+        }
+    }
+}
```

Comment:
we should not be having defaults for presignatures either. In what case is it used? If for testing, we should have a dummy function instead of default

Reply:
Yes, for testing
ok, security first